__tests__/modern.test.ts

```diff
@@ -74,12 +70,8 @@ it('Creates a simple css variable based theme with light and dark', () => {
     `,
     `
       .test {
-        color: var(--color);
-        background-image: linear-gradient(to right, var(--color), var(--color))
-      }
-
-      :root {
-        --color: purple
+        color: var(--color, purple);
+        background-image: linear-gradient(to right, var(--color, purple), var(--color, purple))
```

Comment:
I love how this looks for single usage, but once we get to multiple like here I think it might make more sense for the :root declaration. @kendallgassner whatcha think?